pvdeg/weather.py

```diff
@@ -82,19 +82,28 @@ def get(database, id=None, geospatial=False, **kwargs):
             raise NameError("Weather database not found.")
 
         if "relative_humidity" not in weather_df.columns:
-            print('Column "relative_humidity" not found in DataFrame. Calculating...')
+            print('Column "relative_humidity" not found in DataFrame. Calculating...' , end='')
             weather_df = humidity._ambient(weather_df)
+            print('\r                                                                        ', end='')
+            print('\r', end='')
 
         # map meta-names as needed
         for key in [*meta.keys()]:
             if key in META_MAP.keys():
                 meta[META_MAP[key]] = meta.pop(key)
+        if meta['Source'] == 'NSRDB':
+            meta['Wind_Height_m']=2
```

Comment:
change to lower case to align with pvlib naming convention

Reply:
All the meta data files for the weather data use capitals in all the meta data variable names. I don't think this one should be different.

Reply:
I see that PSM3 is using capitalized names whereas NSRDB is using lowercase. We had build our mapping around the NSRDB/pvlib convention: META_MAP = {"elevation": "altitude", "Local Time Zone": "timezone"}

I do see that we don't have the variable mapping for PSM3 in place yet. I think we should add the variable mapping to PSM3 and make it consistent with the rest of the code.